html/pfappserver/root/src/views/Configuration/mfas/_components/FormTypeAkamai.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <base-form
+    :form="form"
+    :meta="meta"
+    :schema="schema"
+    :isLoading="isLoading"
+  >
+    <form-group-identifier namespace="id"
+      :column-label="$i18n.t('Name')"
+      :disabled="!isNew && !isClone"
+    />
+
+    <form-group-app-id namespace="app_id"
+      :column-label="$i18n.t('The App ID of the Akamai MFA')"
+      :text="$i18n.t('Define the App ID provided by Akamai MFA.')"
```

Comment:
I don't think the 'help' adds much value since its the label with the verb define before it. Might as well remove them where that is the case IMO